tests/Models/TicketTest.php

```diff
@@ -805,4 +803,17 @@ public function checking_if_a_ticket_is_owned_by_a_user ()
         $this->assertTrue($owned->status()->ownedBy($user));
         $this->assertFalse($notOwned->status()->ownedBy($user));
     }
+
+    /** @test */
+    public function it_adds_a_user_to_ignored_list_on_creation ()
```

Comment:
Don't really need this test, this is more testing the test suite than anything else.